rfd/0062-tsh-proxy-template.md

```diff
@@ -126,11 +162,17 @@ is equivalent to the following when connecting to `node-1.leaf1.us.acme.com`:
 tsh proxy ssh -J leaf1.us.acme.com:3080 %r@node-1:3022
 ```
 
-### Auto-login
+#### Auto-login
 
 To further improve the UX, we should make sure that `tsh proxy ssh` command
 does not require users to log into each individual leaf cluster beforehand.
 
 Users should be able to login once to their root cluster and use that
 certificate to set up proxy with the extracted leaf's proxy (similar to proxy
 jump scenario).
+
+### Proxy Templates for `tsh ssh`
+
+Proxy Templates can also be used with `tsh ssh` and have feature parity with `ssh`.
+This means that users can go from using a proxy template to `ssh node-1.leaf1.us.acme.com`
+to `tsh ssh node-1.leaf1.us.acme.com` without making any additional change to the template.
```

Comment:
What's the UX going to be like if you're not logged into the leaf cluster? Will we fetch the certificate automatically?

Reply:
It will function identically to `tsh proxy ssh -J {{proxy}}`, so yes.

Reply:
Cool, let's mention this in the doc.